src/TulsiGenerator/Scripts/bazel_build.py

```diff
@@ -937,7 +939,10 @@ def _InstallArtifact(self, outputs):
         return 600
 
     try:
-      output_data = json.load(open(outputs[0]))
+        # Take the first target, as that's the main artifact we care about
+        # either it's the last path component, or the thing after the colon
+        good_name = (self.targets[0].split('/')[-1]).split(':')[-1]
```

Comment:
Please add a comment here with an example of what this series of splits / array slicing will do.

Reply:
Also couldn't you just split the target by `:` and return the last element? Is splitting by `/` necessary?